rbac/internal/views.py

```
@@ -2991,6 +2992,42 @@ def replicate_updated_workspaces(request):
+    Returns:
+        JSON response indicating the task has been queued
+    """
+    if "since" not in request.GET:
+        return JsonResponse({"field": "since", "detail": 'missing query parameter "since"'}, status=400)
+
+    since = request.GET["since"]
+
+    try:
+        datetime.datetime.fromisoformat(since)
+    except ValueError as e:
+        return JsonResponse({"field": "since", "detail": f"invalid datetime: {str(e)}"}, status=400)
```

Ruff (0.16.0)

[warning] Use explicit conversion flag

Replace with conversion flag

(RUF010)